doc/distributed-run.md

```diff
@@ -0,0 +1,52 @@
+## Running load test
+
+Starting a scenario on multiple nodes is slightly more difficult.
+You need a number of machines that will be actually running the test
+(*slaves*) and one controller machine (*master*, which might be one of the test nodes).
+Another aproach to do it is to use docker containers.
+
+Now instead of `amoc_local` use `amoc_dist` - this will tell amoc to distribute
+and start scenarios on all known nodes (except master).
+
+```erlang
+amoc_dist:do(my_scenario, 1, 100, Opts).
+```
+
+Start `my_scenario` spawning 100 amoc users with IDs from range (1,100) inclusive.
+In this case sessions are going to be distributed across all nodes except master.
+At this moment users are distributed in round-robin fashion - user1 goes to slave1,
```

Comment:
> [...] users are distributed in a round-robin fashion — [...]

The article, and also, if we're using dashes, it should be an emm-dash and not a regular hyphen: https://en.wikipedia.org/wiki/Dash#En_dash_versus_em_dash 😛